### PR TITLE
Update firefox set of files for SEO purpose - bug 1345338

### PR DESCRIPTION
**Pull request by peiying2**
* firefox/android/index.lang
* firefox/channel/index.lang
* firefox/desktop/index.lang
* firefox/family/index.lang
* firefox/ios.lang
* firefox/private-browsing.lang
* firefox/sync.lang

### CODE REVIEW

**Comment by peiying2** [comment]
@flodolo, the strings look so similar, I hope I caught them all and not missed any. A few things to note: 
1). firefox/ios.html and firefox/android/index.html contain 'add_lang_files "firefox/sendto"', but I ended up editing other files. What does sendto.lang contain?  
2). I removed any duplicates within a file (in the updates).  I did not check between files if there are dups of updates.
3). One of the files have the same string for both title and description.  https://github.com/FrancescoSTL/bedrock/blob/56e69cb6cfacd2517d5a573ce77d4e636348faa0/bedrock/firefox/templates/firefox/channel/ios.html#L13 & 21.  I commented on it.  
4).  Some tags were dropped or removed.  I removed tag fx49_remove_hello in the file all together, dropped this one tracking_protection while the string stays.  I assume you can't mark "tag obsolete", you just remove it if no longer used.

**Comment by flodolo** [comment]
Unfortunately this will have to wait a bit, not sure I'll be able to get to it before the end of the week with the mess we're in.

1) It's a widget enabled only for some locales in some page
https://www-dev.allizom.org/styleguide/docs/send-to-device/

3) Good catch.

4) Exactly, you just remove the tag on top of the file, and remove the tag binding from the string itself.

File: en-US/firefox/desktop/index.lang [hunk 1]
```diff
@@ -1,9 +1,16 @@
...
 
 
+## TAG: firefox_title_meta_bug1345338
+# HTML page title
+;Free Web Browser for Mac, Windows, Linux | Firefox
+Free Web Browser for Mac, Windows, Linux | Firefox
+
+
 ## TAG: home_title_update
 # This is the <title> tag
```
Comment (flodolo) [inline en-US/firefox/desktop/index.lang]: Add obsolete note

File: en-US/firefox/family/index.lang [hunk 2]
```diff
@@ -1,23 +1,33 @@
...
+
+
 # Title of the page (HTML meta tag)
 ;Firefox – Desktop browser, Android, iOS, Sync, Private Browsing
 Firefox – Desktop browser, Android, iOS, Sync, Private Browsing
 
 
-## TAG: tracking_protection
 # Obsolete title of the page (do not remove)
 ;Firefox – Desktop browser, Android, iOS, OS, Hello, Sync, Private Browsing
```
Comment (flodolo) [inline en-US/firefox/family/index.lang]: You should remove this string

File: en-US/firefox/family/index.lang [hunk 3]
```diff
@@ -1,23 +1,33 @@
...
 
 
-## TAG: fx49_remove_hello
+## TAG: firefox_title_meta_bug1345338
+# HTML page title
+;Free Web Browser for Android, iOS, Desktop | Firefox
+Free Web Browser for Android, iOS, Desktop | Firefox
+
+
 # Title of the page (HTML meta tag)
```
Comment (flodolo) [inline en-US/firefox/family/index.lang]: Add obsolete note, also to the description (` Learn more about our complete lineup of products for desktop and mobile browsing (and more) that help keep you in control online. Free yourself with Firefox.`)

File: en-US/firefox/family/index.lang [hunk 4]
```diff
@@ -1,23 +1,33 @@
...
 
+## TAG: firefox_title_meta_bug1345338
+# HTML page description
+;Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+Millions of people around the world trust Firefox Web browsers on Android, iOS and desktop computers. Fast. Private. Download now!
+
+
 # Obsolete title of the page (do not remove)
 ;Firefox — Desktop browser, Android, OS, Hello, Sync, Marketplace
 Firefox — Desktop browser, Android, OS, Hello, Sync, Marketplace
```
Comment (flodolo) [inline en-US/firefox/family/index.lang]: You should remove this one as well

File: en-US/firefox/ios.lang [hunk 5]
```diff
@@ -1,21 +1,35 @@
...
 ## URL: https://www-dev.allizom.org/%LOCALE%/firefox/ios/
 
 
+## TAG: firefox_title_meta_bug1345338
+# HTML page title
+;Free Mobile Browser App for iPhone, iPad, iOS | Firefox
+Free Mobile Browser App for iPhone, iPad, iOS | Firefox
+
+
 # Title of the page (HTML meta)
```
Comment (flodolo) [inline en-US/firefox/ios.lang]: Add obsolete note.

File: en-US/firefox/ios.lang [hunk 6]
```diff
@@ -1,21 +1,35 @@
...
 
-;Firefox for iOS
-Firefox for iOS
+## TAG: firefox_title_meta_bug1345338
+# HTML page description
+;Join the millions browsing better on iPhone and iPad. Firefox is fast and private. Install Firefox for iOS now!
+Join the millions browsing better on iPhone and iPad. Firefox is fast and private. Install Firefox for iOS now!
 
 
+# HTML page description
```
Comment (flodolo) [inline en-US/firefox/ios.lang]: Add obsolete note.

File: en-US/firefox/private-browsing.lang [hunk 7]
```diff
@@ -1,12 +1,25 @@
...
 ## URL: https://www-dev.allizom.org/%LOCALE%/firefox/private-browsing
 
 
+## TAG: firefox_title_meta_bug1345338
+# HTML page title
+;Private Browser Protects Your Online Privacy | Firefox
+Private Browser Protects Your Online Privacy | Firefox
+
+
 # HTML Title attribute
```
Comment (flodolo) [inline en-US/firefox/private-browsing.lang]: Add obsolete note

File: en-US/firefox/private-browsing.lang [hunk 8]
```diff
@@ -1,12 +1,25 @@
...
 Firefox Web browser — Private Browsing with Tracking Protection — Mozilla
 
 
+## TAG: firefox_title_meta_bug1345338
+# HTML page description
+;Firefox protects your online privacy and blocks trackers that follow you around the web. Get Firefox for private browsing!
+Firefox protects your online privacy and blocks trackers that follow you around the web. Get Firefox for private browsing!
+
+
 # Meta tag (description)
```
Comment (flodolo) [inline en-US/firefox/private-browsing.lang]: Add obsolete note

File: en-US/firefox/sync.lang [hunk 9]
```diff
@@ -2,14 +2,28 @@
...
 ## URL: https://www-dev.allizom.org/%LOCALE%/firefox/sync/
 
 
+## TAG: firefox_title_meta_bug1345338
+# HTML page title
+;Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+Use Bookmarks, Tabs and Passwords Across Devices | Firefox
+
+
+# HTML page title
```
Comment (flodolo) [inline en-US/firefox/sync.lang]: Add obsolete note.

**Comment by flodolo** [comment]
Opened https://github.com/mozilla/bedrock/pull/4751

**Comment by flodolo** [comment]
And let's hold on to see what comes out of this discussion
https://bugzilla.mozilla.org/show_bug.cgi?id=1345338#c25

**Pull request merged**